src/utils/payout.ts

```diff
@@ -0,0 +1,74 @@
+import { abi as ERC20 } from '@openzeppelin/contracts/build/contracts/ERC20.json'
+import EthersAdapter from '@gnosis.pm/safe-ethers-lib'
+import Safe from '@gnosis.pm/safe-core-sdk'
+import { SafeTransactionDataPartial } from '@gnosis.pm/safe-core-sdk-types'
+import * as ethers from 'ethers'
+
+export type PayoutParams = {
+	network?: number;
+	from: string;
+	amount: string;
+	token: string; // pass 0x00 to send ether
+	to: string;
+  execute?: boolean;
+  account?: number;
+  gwei?: number;
+}
+
+export const providerKeyByChainId = new Map<number, string>([
+  [1, process.env.RPC1_ALCHEMY_KEY as string],
+  [5, process.env.RPC5_ALCHEMY_KEY as string], // goerli - safe compatable
+])
+
+export const payout = async (args: PayoutParams) => {
+  // collect baseline info
+  const tokenAddress = ethers.utils.getAddress(args.token)
+  const isEth = tokenAddress === ethers.constants.AddressZero
+  const destinationAddress = ethers.utils.getAddress(args.to)
+  const mnemonic = process.env.MNEMONIC as string
+  const network = args.network || 1
```

Comment:
we should also check that we allow the network and that we have the corresponding alchemy key

Reply:
Also nice detail.